pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -5,11 +5,18 @@ echo "setting ssh password authentication"
 sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
 systemctl reload sshd.service
 
-echo "running RHUI fix"
-yum update -y --disablerepo='*' --enablerepo='rhui-microsoft-azure*'
+#Adding retry logic to yum commands in order to avoid stalling out on resource locks
+for attempt in {1..5}; do
+  echo "running RHUI fix"
```

Comment:
nit: we should echo this line outside of the loop

Reply:
Agreed. Changed and ran make generate